frontend/pages/ManageControlsPage/OSSettings/cards/CustomSettings/components/ConfigProfileStatusTable/ConfigProfileStatusTableConfig.tsx

```diff
@@ -0,0 +1,106 @@
+import React from "react";
+import { Column } from "react-table";
+
+import StatusIndicatorWithIcon from "components/StatusIndicatorWithIcon";
+import {
+  INumberCellProps,
+  IStringCellProps,
+} from "interfaces/datatable_config";
+import { IGetConfigProfileStatusResponse } from "services/entities/config_profiles";
+import ConfigProfileHostCountCell from "../ConfigProfileHostCountCell";
+
+type IConfigProfileStatus = "verified" | "verifying" | "pending" | "failed";
```

Comment:
Nit: consider using this existing [type](https://github.com/fleetdm/fleet/blob/0b72f6c807f07e8701c7971e4dbbd3661956c5db/frontend/interfaces/mdm.ts#L117-L118)